code/game/machinery/vending/vendor_types/squad_prep/squad_engineer.dm

```diff
@@ -205,7 +205,8 @@ GLOBAL_LIST_INIT(cm_vending_clothing_engi, list(
 		/obj/item/explosive/plastic,
 		/obj/item/stack/sandbags_empty = 25,
 		/obj/item/stack/sheet/metal/large_stack,
-		/obj/item/stack/sheet/plasteel/medium_stack,
+		/obj/item/stack/sheet/plasteel/medium_stack = 40,
```

Comment:
Wouldn't this instead just be `/obj/item/stack/sheet/plasteel/med_large_stack`

Reply:
That would appear to also be the same value.
I just did a built with that type. 

I did the number because I based my change on the sandbag line on line 206. 

Pushing that edit now to provide code consistency.